UIDev/UITest.cs

```diff
@@ -148,6 +148,6 @@ private static string FindGameDataPath()
                 }
             }
         }
-        return "D:\\installed\\SquareEnix\\FINAL FANTASY XIV - A Realm Reborn\\game\\sqpack";
+        return "C:\\M.2 Steam Library\\steamapps\\common\\FINAL FANTASY XIV Online\\game\\sqpack";
```

Comment:
Local change that doesn't belong to this PR?

BTW I've now grabbed more recent version of this code from xivlauncher that accounts for steam version, so this should no longer be needed for you.